clients/cli/nmcli.c

```diff
@@ -355,8 +560,17 @@ set_colors (NmCli *nmc, NmcColorOption *color_option)
 		nmc->nmc_config_mutable.use_colors = FALSE;
 		break;
 	}
+
+	if (nmc->nmc_config_mutable.use_colors && nmc->color_scheme) {
+		if (!parse_color_scheme (nmc, &error)) {
+			g_printerr (_("Error parsing color scheme: %s"), error->message);
```

Comment:
Does this really warrant a warning?

Reply:
I think so. Not sure. In general, when you mess up your configuration file and it doesn't work, this could help you find out what's wrong.